aviatrix/resource_aviatrix_account.go

```diff
@@ -396,11 +428,20 @@ func resourceAviatrixAccountCreate(ctx context.Context, d *schema.ResourceData,
 		if account.AwsgovAccountNumber == "" {
 			return diag.Errorf("aws gov account number needed for aws gov cloud")
 		}
-		if account.AwsgovAccessKey == "" {
-			return diag.Errorf("aws gov access key needed for aws gov cloud")
-		}
-		if account.AwsgovSecretKey == "" {
-			return diag.Errorf("aws gov secret key needed for aws gov cloud")
+		if awsGovIam {
+			if account.AwsgovRoleApp == "" {
+				account.AwsgovRoleApp = fmt.Sprintf("arn:aws-us-gov:iam::%s:role/aviatrix-role-app", account.AwsChinaAccountNumber)
+			}
+			if account.AwsgovRoleEc2 == "" {
+				account.AwsgovRoleEc2 = fmt.Sprintf("arn:aws-us-gov:iam::%s:role/aviatrix-role-ec2", account.AwsChinaAccountNumber)
```

Comment:
why are setting AWS China account number for AWS Gov IAM?

Reply:
Its a typo as well